section4/ex2_sec4.rb

```diff
@@ -0,0 +1,56 @@
+#Exercises:
+#1. Create a class called MyCar.
+
+class MyCar
+  attr_accessor :color
+  attr_reader :year
+
+  def initialize(year, color, model)
+    @year = year
+    @color = color
+    @model = model
+    @speed = 0
+  end
+
+  def speeds(number)
+    @speed += number
+    puts "Going too fast!"
+  end
+
+  def brakes(number)
+    @speed -= number
+    puts "Keep slowing down"
+  end
+
+  def speed
+    puts "You are going #{@speed} miles per hour."
+  end
+
+  def shut_car_off
+    @speed = 0
+    puts "Please turn the car off"
+  end
+
+  def spray_paint(color)
+    self.color = color
```

Comment:
What is the reason you used `self` here??

Reply:
Because we already have the attr_accessor :color.